quickstart.md

```diff
@@ -74,3 +74,5 @@ interesting.
 Also, subscribe to our
 [perspectiveapi-announce@ Google Group](https://groups.google.com/forum/#!forum/perspective-announce/join) to
 get notified when we make changes to the API.
+
+For support and to contact us, see: https://support.perspectiveapi.com/s/. 
```

Comment:
nit: could you remove the "s/" part of the URL, so it's just `https://support.perspectiveapi.com/` like elsewhere in the documentation? :-)